src/mrpro/data/_CsmData.py

```diff
@@ -68,7 +70,7 @@ def _iterative_walsh_csm(
     def from_idata_walsh(
         cls,
         idata: IData,
-        smoothing_width: SpatialDimension[int] = SpatialDimension(5, 5, 5),
+        smoothing_width: int | SpatialDimension[int] = 5,
         niter: int = 3,
```

Comment:
niter-> power_iterations

docstring: number of iterations used to determine dominand eigenvector

Reply:
changed to `power_iterations` for now but not sure if I would prefer `n_iterations`...

Reply:
I will leave it as `power_iterations`